.gitignore

```diff
@@ -30,3 +30,4 @@ packages/k6/src/config/http.json
 # Secrets
 .envrc
 .env
+.env.sh
```

Comment:
What is this?

Reply:
If we want to update this I would actually prefer to combine this with the previous line with something like `.env*`

Reply:
I will take this out. The problem I was facing was that if i used a .env file, my variables would not be read by the EA. On my local machine (Linux), I instead had to set these as variables through my command line, and only then would my variables be passed through. This is why you see a "dotenv" line of code as i was trying to test why the .env would not be read

Reply:
This is for local testing? Try 


yarn build && yarn start --env-file=/Users/abcde/Documents/env_file